frontiers_brsu_building_c/models/objects/adversarial-plugin-states.json

```diff
@@ -0,0 +1,56 @@
+{
+    "@context": [
+        {
+            "pl": "https://secorolab.github.io/metamodels/plugin#",
+	    "door": "https://secorolab.github.io/models/door#"
+        },
+        "https://comp-rob2b.github.io/metamodels/geometry/spatial-relations.json",
+        "https://comp-rob2b.github.io/metamodels/geometry/structural-entities.json",
+        "https://comp-rob2b.github.io/metamodels/geometry/coordinates.json",
+        "https://comp-rob2b.github.io/metamodels/kinematic-chain/structural-entities.json",
+        "https://secorolab.github.io/metamodels/floor-plan/object.json",
+        "https://secorolab.github.io/metamodels/geometry/polytope.json",
+        "https://secorolab.github.io/metamodels/floor-plan/state.json",
+        "http://localhost:8000/floor-plan/plugin.json",
```

Comment:
there seems to be a few `localhost` in several models

Reply:
cleaned them up